tests/src/Kernel/Update405Test.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Drupal\Tests\lightning\Kernel;
+
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\lightning\Update\Update405;
+
+/**
+ * @group lightning
+ */
+class Update405Test extends KernelTestBase
+{
+    protected static $modules = array('system');
+
+    /**
+     */
+  public function testEnableAutosaveForm() {
```

Comment:
The doc comment of this method should probably be `@covers ::enableAutosaveForm`